Source/WebCore/loader/FTPDirectoryParser.cpp

```diff
@@ -168,8 +168,7 @@ FTPEntryType parseOneFTPLine(const char* line, ListState& state, ListResult& res
                         state.listStyle = lstyle = 'E';
 
                         p = &(line[lineLenSansWsp]);
-                        result.filename = tokens[1];
-                        result.filenameLength = p - tokens[1];
+                        result.filename = std::span(tokens[1], p - tokens[1]);
```

Comment:
Isn't there a `span` constructor that takes two pointers? Or am I imagining things?

Reply:
Oh, you're right! I'll update.